libcloud/compute/drivers/gce.py

```diff
@@ -6058,9 +6156,25 @@ def deploy_node(self, name, size, image, script, location=None,
         metadata = {'items': [{'key': 'startup-script', 'value': script_data}]}
 
         return self.create_node(name, size, image, location=location,
-                                ex_network=ex_network, ex_tags=ex_tags,
-                                ex_metadata=metadata,
-                                ex_service_accounts=ex_service_accounts)
```

Comment:
I would simply add `, **kwargs` to the arguments.

This means old uses of `deploy_node` will continue to work, and at the same time we will automatically support new `create_node` arguments.